src/app/client/src/app/modules/core/services/search/search.service.ts

```diff
@@ -268,6 +268,29 @@ export class SearchService {
     }
     return this.publicDataService.post(option);
   }
+
+  /**
+   * Content Search using dail code.
+   *
+   * @param {SearchParam} requestParam api request data
+  */
+ contentDialCodeAssembleSearch(requestParam: SearchParam, addDefaultContentTypesInRequest: boolean = true):
+ Observable<ServerResponse> {
+ const option = {
+   url: this.config.urlConFig.URLS.DAIL_ASSEMBLE_PREFIX,
+   data: {
+     request: {
+      source: 'web',
```

Comment:
you need to create page api for source: 'web',

Reply:
Ok I will sync up with KP team and get create a page in DEV for `web`